### PR TITLE
WEB-4866: Registration Updates

### PR DESCRIPTION
**Pull request by DarrenHooper**

### CODE REVIEW

File: scss/controls/_spinner.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+$offset: 280;
```
Comment (mctaggaj) [inline scss/controls/_spinner.scss]: Move this out of controls

File: scss/controls/_spinner.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+$duration: 1s;
```
Comment (mctaggaj) [inline scss/controls/_spinner.scss]: Global variables are bad

File: scss/inputs/_text_field.scss [hunk 3]
```diff
@@ -1,11 +1,12 @@
 @mixin _text_field {
```
Comment (mctaggaj) [inline scss/inputs/_text_field.scss]: move into controls

File: scss/mixins/_typography.scss [hunk 4]
```diff
@@ -6,30 +6,33 @@
 
 @mixin uikit-heading1 {
   @include _uikit-typography($uikit-size-heading1);
-  font-family: $uikit-font-face-alternate;
+  font-family: $uikit-font-face-default;
+  font-weight: 500;
```
Comment (mctaggaj) [inline scss/mixins/_typography.scss]: should be variable

File: scss/typography/_typography.scss [hunk 5]
```diff
@@ -35,4 +35,6 @@
 
 body {
   @include uikit-regular;
+  -webkit-font-smoothing: antialiased;
```
Comment (mctaggaj) [inline scss/typography/_typography.scss]: use: https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
The compilation process will do the reset

File: scss/controls/_checkbox.scss [hunk 6]
```diff
@@ -44,3 +48,35 @@
...
+  @extend .#{$uikit-prefix}checkbox__label;
+  font-size: $uikit-size-small;
+  font-weight: $uikit-font-weight-default;
+  &:before {
+    float: left;
+    margin-right: $uikit-spacing-normal;
+    margin-left: -4px; // override bootstrap 4px on checkbox
+  }
+  &:after {
+    content: url('/common/public/images/icons/checkmark-white.svg');
```
Comment (mctaggaj) [inline scss/controls/_checkbox.scss]: Fix url

File: scss/controls/_checkbox.scss [hunk 7]
```diff
@@ -44,3 +48,35 @@
...
+      }
+      &:after {
+        opacity: 1;
+        transition-delay: 0.3s;
+      }
+    }
+  }
+}
+
+.#{$uikit-prefix}checkbox__label--alternate {
```
Comment (mctaggaj) [inline scss/controls/_checkbox.scss]: Why is this alternate?

File: scss/lists/_checklist__cell.scss [hunk 8]
```diff
@@ -0,0 +1,17 @@
+.#{$uikit-prefix}checklist__cell {
```
Comment (mctaggaj) [inline scss/lists/_checklist__cell.scss]: one file

File: scss/controls/_button.scss [hunk 9]
```diff
@@ -42,6 +43,9 @@
 
 
 // Size modifiers
+.#{$uikit-prefix}button--large {
```
Comment (mctaggaj) [inline scss/controls/_button.scss]: This naming should be confirmed with design.
(Wide may make more sense)

File: scss/controls/_button.scss [hunk 10]
```diff
@@ -85,6 +89,17 @@
   }
 }
 
+.#{$uikit-prefix}button--loading {
+  min-height: 48px;
```
Comment (mctaggaj) [inline scss/controls/_button.scss]: can we just set the text colour to transparent?

File: scss/controls/_button.scss [hunk 11]
```diff
@@ -85,6 +89,17 @@
...
+  cursor: not-allowed;
+}
+
+.#{$uikit-prefix}button--loading__spinner {
+  @extend .#{$uikit-prefix}spinner;
```
Comment (mctaggaj) [inline scss/controls/_button.scss]: Can this be accomplished in the after?

File: scss/controls/_spinner.scss (hunk 1 above)
Comment (DarrenHooper) [reply scss/controls/_spinner.scss]: 👍

File: scss/inputs/_text_field.scss (hunk 3 above)
Comment (DarrenHooper) [reply scss/inputs/_text_field.scss]: 👍

File: scss/controls/_checkbox.scss (hunk 7 above)
Comment (DarrenHooper) [reply scss/controls/_checkbox.scss]: 👍

File: scss/lists/_checklist__cell.scss (hunk 8 above)
Comment (DarrenHooper) [reply scss/lists/_checklist__cell.scss]: 👍

File: scss/controls/_checkbox.scss (hunk 6 above)
Comment (DarrenHooper) [reply scss/controls/_checkbox.scss]: 👍

File: scss/typography/_typography.scss (hunk 5 above)
Comment (DarrenHooper) [reply scss/typography/_typography.scss]: Turns out, no it won't. You can't specify those settings, only "auto" or "always".

File: scss/mixins/_typography.scss (hunk 4 above)
Comment (DarrenHooper) [reply scss/mixins/_typography.scss]: 👍

File: scss/controls/_spinner.scss (hunk 2 above)
Comment (DarrenHooper) [reply scss/controls/_spinner.scss]: 👍

File: scss/controls/_button.scss (hunk 9 above)
Comment (DarrenHooper) [reply scss/controls/_button.scss]: 👍

File: scss/controls/_button.scss (hunk 11 above)
Comment (DarrenHooper) [reply scss/controls/_button.scss]: https://jira.enflick.com/browse/WEB-4866

File: scss/controls/_button.scss (hunk 10 above)
Comment (DarrenHooper) [reply scss/controls/_button.scss]: Will be done as part of https://jira.enflick.com/browse/WEB-4866

File: scss/typography/_typography.scss (hunk 5 above)
Comment (mctaggaj) [reply scss/typography/_typography.scss]: 👍

**Pull request merged**